index.html

```diff
@@ -0,0 +1,216 @@
+                </li>
+                <li class="portfolio-link">
+                    <a class="portfolio-link-a new-link" href="#contact-section">Join</a>
+                </li>
+                <li class="portfolio-link ">
+                    <a class="portfolio-link-a new-link" href="#contact-section">News</a>
+                </li>
+            </ul>
+        </div>
+        <!-- ================First Section Home page =========================-->
+        <article class="main-section-article">
+            <h1>Sharm el-Sheikh Climate Change Conference</h1>
+            <h2>A defining moment in the fight against climate change 2022</h2>
+            <p class="article-1">Science has established beyond doubt that the window for action is closing rapidly. In
+                November 2022,
+                Egypt will host the 27th Conference of the Parties of the UNFCCC (COP27) in Sharm El-Sheikh, with a view
+                to building on previous successes and paving the way for future ambition.
+                A golden opportunity for all stakeholders to rise to the occasion and tackle effectively the global
+                challenge of climate change facilitated by Egypt on the African continent.</p>
+            <p class="date">6 November to 18 November 2022</p>
+            <p class="place">@ Sharm El Sheikh International Convention Centre</p>
+        </article>
+    </section>
+    <!--============ Second Section Home page =======================-->
+    <section class="second-section">
+        <h2>Main Programe</h2>
+        <hr class="hr-about-1">
+        <div class="container">
+            <div class="card">
+                <img src="img/program_icon_01.png" alt="icon-1">
+                <p class="title">Lecture</p>
+                <p class="content">lesten to the spokers from various countries about climate Change</p>
+            </div>
+            <div class="card">
+                <img src="img/program_icon_02.png" alt="icon-1">
+                <p class="title">Extensive experience</p>
+                <p class="content">Helping transform to achieve adaptation and resilience</p>
+            </div>
+            <div class="card">
+                <img src="img/program_icon_03.png" alt="icon-1">
+                <p class="title">Expertise</p>
+                <p class="content">To produce industry-wide progress on sustainability goals.</p>
+            </div>
+            <div class="card">
+                <img src="img/program_icon_04.png" alt="icon-1">
+                <p class="title">partnership </p>
+                <p class="content">With public- and private-sector leaders as well</p>
+            </div>
+            <div class="card">
+                <img src="img/program_icon_05.png" alt="icon-1">
+                <p class="title">catalyze</p>
+                <p class="content">Ability to catalyze climate innovation by mobilizing funding across</p>
+            </div>
+        </div>
```

Comment:
- [ ] Please make this semantic by wrapping this in a `ul` tag and each program in a `li` tag.